docs/src/tutorials/algorithms/tsp_lazy_constraints

```diff
@@ -0,0 +1,286 @@
+
+# Copyright (c) 2019 Arpit Bhatia and contributors                               #src
+#                                                                                #src
+# Permission is hereby granted, free of charge, to any person obtaining a copy   #src
+# of this software and associated documentation files (the "Software"), to deal  #src
+# in the Software without restriction, including without limitation the rights   #src
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      #src
+# copies of the Software, and to permit persons to whom the Software is          #src
+# furnished to do so, subject to the following conditions:                       #src
+#                                                                                #src
+# The above copyright notice and this permission notice shall be included in all #src
+# copies or substantial portions of the Software.                                #src
+#                                                                                #src
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     #src
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       #src
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    #src
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         #src
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  #src
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  #src
+# SOFTWARE.     
+
+
+# ## [Traveling Salesperson Problem (via callbacks)](@id tsp_lazy)
+# 
+# **Originally Contributed by**: Daniel Schermer
+# 
+# This notebook describes how to implement the Traveling Salesperson Problem in JuMP using lazy constraints that dynamically separate subtours.
+# To be more precise, we use lazy constraints to cut off feasible subtours only when necessary and not before needed.
+# The model has been tested with Julia Version 1.7.0, JuMP.jl Version 0.22.1, Gurobi.jl Version 0.10.1 (Solver Version 9.5), and GLPK Version 0.15.2.
```

Comment:
remove gurobi

Reply:
Removed! The code is now purely based on using GLPK as a solver.